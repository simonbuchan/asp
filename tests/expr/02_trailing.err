UNPARSED:
1: op semicolon ';'
