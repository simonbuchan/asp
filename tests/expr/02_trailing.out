(minus (plus 1 (mul 2 3)) 4)
